Tasks/AzureRmWebAppDeployment/KuduUtility.ts

```diff
@@ -0,0 +1,89 @@
+import Q = require('q');
+import tl = require('vsts-task-lib/task');
+import path = require("path");
+import fs = require("fs");
+var request = require ('request');
+var archiver = require('archiver');
```

Comment:
Applied OSS licence for 1.01 and I'm using that. But need to update thirdpart oss document and Source scanned version or not need to confirm